instagram-page-react/src/pages/profilePage/organisms/posts/Posts.js

```diff
@@ -58,8 +61,10 @@ function Posts(props) {
     );
 }
 
+const memoizedReverseArray = defaultMemoize(reverseArray);
```

Comment:
Should we defaultMemoize this function and why?

If yes, should we defaultMemoize this in the component or in the selector?

Reply:
Have defaultMemoized the function here in the selector, as any other component in future may require this functionality, so memoizing the function in one place would be better in that case.